### PR TITLE
Feature/#18 add headers in requests

### PR DESCRIPTION
**Pull request by pgarciaegido**
Adds headers in the `/status` request, and query params `clientName` and `clientVersion` to pass to the checkout url.

### CODE REVIEW

File: packages/ui/react-ui/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
         "access": "public"
     },
     "scripts": {
+        "prebuild": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
```
Comment (pgarciaegido) [inline packages/ui/react-ui/package.json:27]: creates a `version.ts` file where to get the package.json version

**Review by alfonso-paella — APPROVED** [review]
LGTM. 

One thing I dont understand is how does the version number update with lerna. Make sure to test that that process doesn't break what you put together here

**Comment by pgarciaegido** [comment]
@alfonso-paella you mean with the `publish` command? I guess that when you bump a package version, you would need to build and then publish. If the `postbuild` hook works as expected on the release process, I would assume that the `prebuild` would work as well.

How do you normally handle the release process? Maybe @mPaella knows too.

Thanks!

**Review by alfonso-paella — APPROVED** [review]
Thanks for addressing that

**Pull request merged**